unifi/DOCS.md

```diff
@@ -86,13 +86,11 @@ the normal Home Assistant methods (e.g., using Samba, Terminal, SSH).
 ## Migrating from an existing controller
 
 If you want to migrate from an existing controller to the controller provided
-by this addon, Ubiquiti has an excellent tutorial on this:
+by this addon, Ubiquiti has instructions on how to back up your old controller and restore the configuration:
 
-<https://help.ubnt.com/hc/en-us/articles/115002869188>
+<https://help.ui.com/hc/en-us/articles/204952144-UniFi-How-to-Create-and-Restore-a-Backup>
 
-This article explains in detail how to use the UniFi Site Export Wizard
-to quickly and easily export sites from one Controller
-(including configuration and devices) to another (e.g., this add-on).
+Should the above link no longer be available, please search the [Unifi support pages][ui-help] for backing up & restoring your controller.
```

Comment:
Hmmm, that sounds not like something we should document this way.

If the link is no longer available, we need to fix the documentation, just like you did now 😉 

Let's revert this change.